src/index.ts

```diff
@@ -78,6 +78,7 @@ export type { StringModule } from './modules/string';
 export type { SystemModule } from './modules/system';
 export type { VehicleModule } from './modules/vehicle';
 export type { WordModule } from './modules/word';
+export type { mergeLocales } from './utils/merge-locales';
```

Comment:
Oh wait, was it intended to export only the type of the function instead of the real function?

I think you wanted this:
```suggestion
export { mergeLocales } from './utils/merge-locales';
```